examples/otlp/src/main/java/io/opentelemetry/example/otlp/ExampleConfiguration.java

```diff
@@ -41,7 +41,11 @@ static OpenTelemetry initOpenTelemetry() {
     SdkTracerProvider tracerProvider =
         SdkTracerProvider.builder()
             .addSpanProcessor(spanProcessor)
-            .setResource(AutoConfiguredOpenTelemetrySdk.initialize().getResource())
```

Comment:
This was causing an error because it sets `GlobalOpenTelemetry` by default, which is set again on line 47 (51 in the revised code) via `OpenTelemetrySdk.builder().setTracerProvider(tracerProvider).buildAndRegisterGlobal()`.